uber/templates/groups/form.html

```diff
@@ -196,13 +196,14 @@
             <textarea rows="3" cols="30" name="special_needs">{{ group.special_needs }}</textarea>
         </div>
     </div>
```

Comment:
Yeah.. I think my preference here would be to have two sets of Amazon-style address fields, one of which is filled with the other if you check a "same as other address" checkbox. That's outside the scope of this for right now, though.